app/controllers/frontend_log_controller.rb

```diff
@@ -46,6 +46,8 @@ class FrontendLogController < ApplicationController
   # rubocop:enable Layout/LineLength
```

Comment:
I tested this manually and I did see the expected results! 

But I do have a couple questions.

1. Is the reason to do this only on the back image because:
    1. it'll be in both biometric and non-biometric
    2. it will return the same resolution for front, back, and selfie anyway so we don't need all three?

 2. > Is this worth it? It sends a flurry of logs and I could see this having performance impacts.
    1. I counted 7 `idv_camera_info_logged ` events for each upload. The logs have a lot of info so I could be missing it, but I don't see any unique info between the 7 events. That...does seem like a lot for one log, and it seems hard to sort through. You have a good point on potential performance impact as well.
    2. I think ideally if we could cut it down to one event per capture, then it'd be worth it. But 7 events per image upload might be a lot to both sort through, and possibly costly in performance and maybe even AWS cost wise.
    4. I do wonder if it's hitting the event that often because of using `hasStartedCropping`. Maybe that switches from true to false enough to cause the multiple hits?
    5. What if we used a different event to trigger it? Instead of putting it in the `AcuantCamera` itself, could we trigger it, for example, at the same time as `idv_back_image_added` and `idv_image_capture_failed` if source === SDK?
    6. If what's in `iv` is impossible or would not solve the issue, I would be curious what product folks think about the tradeoff.

Reply:
Thanks! I've fixed this issue by basically deduplicating the logs before sending them. I now get one log from my phone, but I left open the possibility that there will be multiple cameras (and multiple logs) from other phones. Can you re test?

Reflecting: I've been trying to keep the scope manageable on this ticket, but I think I've been doing that by leaving out things that are, in fact necessary ( like the hook and this event de duplication). Lesson learned.

Reply:
The reason to do this on the back image is that we only need to check the camera resolution once per user. This code looks at -all- of the cameras that a phone has, so we only need to do it once. I chose the back camera as the place to do it b/c its in all flows. "Front" would work fine too.

I think the more in depth answer to your question is that as far as I can tell the phone doesn't return any particularly good information about the different sets of cameras it has. Like it just says "here's all the cameras I have" and a bunch of data that seems to be the same across all cameras? I assume there's a good reason for this and that it's different per phone.

Reply:
Thank you for addressing these Charley!

That reasoning makes sense to me. I am surprised that the data is the same between cameras, because I haven't felt that pics taken with the front facing camera are as good as the back facing camera on my iphone11.

But I can't find a definitive source that they should be different, and "feeling" is subjective.

Reply:
On another note - I did test again! Two things:

1. It worked mostly as expected, and this time with only one event!
2. It does work differently than the other events still.
    1. The difference: `idv_camera_info_logged` is only logged once - even if I retake the back photo. This is different behavior than other back photo only events like `Frontend: IdV: back image clicked` and `Frontend: IdV: back image added`. Those events occur each time you take a back image photo. But the only way I could re-trigger `idv_camera_info_logged` was to re-load the page and take another back photo
    2. I don't think this is necessarily a blocker, but perhaps worth noting to product folks.

Reply:
Another note, apologies I didn't get to fully review the code yet so can't approve. Was hoping to today but ran out of time. Instead focused on manual testing, but can review the code tomorrow.